week1/main.js

```diff
@@ -0,0 +1,65 @@
+// This is a class based component that has two props that are reuseable
+// as demonstrated in lines 47 through 49.
+class ListItems extends React.Component {
```

Comment:
`ListItem` would be a better name I think, since it's only one item of the list.

Reply:
Hi @ArcoMul,
Thanks for your time and for the feedback. The changes have been made.